back-end/database/db.sql

```diff
@@ -14,13 +15,23 @@ CREATE TABLE Secrets (
   FOREIGN KEY (user) REFERENCES Users(user_id)
 );
 
-INSERT INTO Users (username, password)
+INSERT INTO Users (username, password, salt)
 VALUES
-  ("admin", "password"),
-  ("bob8", "p@ssw0rd"),
-  ("andreP3000", "3l!teHaxor"),
-  ("shyltontor123", "Brazillionaire"),
-  ("chungusamongus", "password123");
+  ("admin",
+  "97d0ff0c7c8a9a4583772c9ea405314c33aa5465355c952f55f2d4983f6306dd8e7c3c96e09e6b6712775352e952c7afabf860731651644f8b5e36cdd9113880",
+  "samplesalt1"),
+  ("bob8",
```

Comment:
I couldn't login using: p@ssw0rd

Reply:
Uh oh. I may have hashed it wrong, cuz I did it manually haha